src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -198,6 +198,7 @@
 Outbound_Connectivity_Check_Failed = "Outbound network connectivity check failed"
 DNS_Check_Failed = "DNS Resolution failed"
 Cluster_Diagnostic_Prechecks_Failed = "Cluster diagnostic prechecks failed"
+Cluster_Diagnostic_Prechecks_Incomplete = "Cluster diagnostic prechecks did not complete"
```

Comment:
Nit thought: should we say failed to complete . Did complete might sound like an issue with our prechecks and not CX env issue